### ISSUE TITLE
Detection Bypass?

### ISSUE THREAD

**Issue by kevosmin** [BandarHL/BHTikTok#24]
My likes and follows don’t work when I use BHtiktok, it works fine on Stock TikTok from the AppStore

**Comment by lockieluke** [comment]
Yeah same here is there a maintained fork of this, it hasn't been updated since last year
